CRM/Admin/Page/Extensions.php

```diff
@@ -219,6 +221,16 @@ public function formatLocalExtensionRows() {
           $row['id']
         );
       }
+      // Determine if this is a core extension or custom extension
+      // Core extensions are in the main CiviCRM directory, custom are in extensionsDir
+      if (str_contains($row['path'], $config->extensionsDir)) {
```

Comment:
I really like how the layout and pages work. It's a nice improvement.

On windows this path comparison will likely be inconsistent. Suggest converting backslashes to forward slashes, or maybe there's another way to tell if an extension is core or not without using paths (like author="CiviCRM" ?)
```suggestion
      if (str_contains(str_replace('\\', '/', $row['path']), str_replace('\\', '/', $config->extensionsDir))) {
```

Reply:
ok, i will do required changes

Reply:
FWIW, there's a helper `CRM_Utils_File::isChildPath()` which does this.